.taskcluster.yml

```diff
@@ -178,11 +126,22 @@ tasks:
               env:
                 TASKCLUSTER_SECRET: project/relman/rust-code-analysis/deploy
               command:
+                - ARTIFACT_ONE=rust-code-analysis-linux-x86_64.tar.gz
+                - ARTIFACT_TWO=rust-code-analysis-cli.exe
+                - ARTIFACT_TWO_ZIP=rust-code-analysis-windows-x86_64.zip
+                - taskboot
+                - retrieve-artifact
+                - "--output-path=/tmp"
+                - "--artifacts=$ARTIFACT_TWO:public/$ARTIFACT_TWO"
+                - cd /tmp
+                - strip $ARTIFACT_TWO
+                - zip $ARTIFACT_TWO_ZIP $ARTIFACT_TWO
                 - taskboot
                 - github-release
                 - mozilla/rust-code-analysis
                 - "${version}"
-                - "--asset=rust-code-analysis-linux-x86_64.tar.gz:public/rust-code-analysis-linux-x86_64.tar.gz"
+                - "--asset=$ARTIFACT_ONE:public/$ARTIFACT_ONE"
+                - "--asset=$ARTIFACT_TWO_ZIP:/tmp/$ARTIFACT_TWO_ZIP"
```

Comment:
I think `taskboot github-release` will try to take the assets from a dependency, we might have to add a `--local-asset` argument to support this.

Reply:
I guess we could strip in an intermediate task and then upload from there using the `--asset` for now. What do you think?